libioc/Config/Jail/Globals.py

```diff
@@ -88,7 +88,7 @@
     "exec_timeout": "600",
     "stop_timeout": "30",
     "mount_procfs": "0",
-    "mount_devfs": "1",
+    "mount_devfs": "0",
```

Comment:
oh?

can you explain this change?

Reply:
The default property `allow_mount_devfs` on iocage was `0`, while `mount_devfs` was `1`. Combined, the jail is not allowed to mount it's own devfs.

```console
$ ioc create myjail allow_mount_devfs=no mount_devfs=yes
myjail successfully created from 12.0-RELEASE!
$ ioc start myjail
[-] JailStart@myjail: ...
  [+] JailDependantsStart@myjail: No dependant jails [0.0s]
  [+] JailResolverConfig@myjail: OK [0.001s]
  [+] JailResourceLimitAction@myjail: disabled [0.001s]
  [+] JailHookPrestart@myjail: SKIPPED [0.0s]
  [+] BasejailStorageConfig@myjail: OK [0.024s]
  [+] MountFstab@myjail: OK [0.0s]
  [+] JailAttach@myjail: OK [0.021s]
  [+] JailHookCreated@myjail: SKIPPED [0.0s]
  [-] MountDevFS@myjail: FAILED [0.047s]
Failed to mount /dev: mount: devfs: Operation not permitted
```

Because of that, either of the defaults needed be changed. Choosing the lower permissions as default, appears to be the right choice here. Otherwise the default `enforce_statfs` would be `1`, not `2` as defined in the Globals.